### PR TITLE
Detemplate for CSES Restaurant Customers (2nd sol)

### PR DESCRIPTION
**Pull request by tachithanhdanh**
Remove the template for the 2nd implementation which was mentioned in [this thread](https://github.com/cpinitiative/usaco-guide/issues/806)
I also fixed a minor mistake in the visualization

### CODE REVIEW

**Review by frodakcin — APPROVED** [review]
Code passes, yes?

**Comment by tachithanhdanh** [comment]
> Code passes, yes?

yes

**Comment by tachithanhdanh** [comment]
Ready to be merged

File: solutions/CSES Restaurant Customers.mdx [hunk 1]
```diff
@@ -208,24 +192,25 @@ int main() {
...
 	int l = 0;
-	vt<long long> pfx(2*N + 1); // prefix sum array
-	vt<int> arr(2*N);
+	vector<long long> pfx(2*N + 1); // prefix sum array
+	vector<int> arr(2*N);
 
-	F0R(i, 2*N) { // coordinate compression
+	for (int i = 0; i < 2*N; ++i) { // coordinate compression
 		if(i == 0) { customers[i].first = 0; }
 		else if(customers[i].first > curr) { l++; curr = customers[i].first; } // we move a pointer
```
Comment (frodakcin) [inline solutions/CSES Restaurant Customers.mdx:200]: Ack this line is very long but I can't be bothered to fix it

**Review by frodakcin — COMMENTED** [review]
Don't think it matters enough... especially if we had hiding TOC feature

**Pull request merged**